.github/CODEOWNERS

```diff
@@ -22,13 +26,23 @@
 /libraries/Adapters/**              @garypretty
 
 # Adaptive Expressions
-/libraries/AdaptiveExpressions/**   @tomlm @carlosscastro @boydc2014 @chrimc62
+/libraries/AdaptiveExpressions/**   @microsoft/bf-adaptive @microsoft/bb-dotnet @microsoft/bf-admin
 
 # Platform Integration Libaries
-/libraries/integration/Microsoft.Bot.Builder.Integration.ApplicationInsights.Core/**           @garypretty
-/libraries/integration/Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi/**         @garypretty
-/libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/**                        @johnataylor
-/libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.WebApi/**                      @johnataylor
+/libraries/integration/Microsoft.Bot.Builder.Integration.ApplicationInsights.Core/**           @garypretty @microsoft/bb-dotnet @microsoft/bf-admin
```

Comment:
Do we need to repeat @microsoft/bb-dotnet @microsoft/bf-admin? They are defined as global. If we do, then we probably need to repeat them in things like Adapters or functional tests.

Reply:
I don't think we need to repeat them. It seems a bit redundant. I would consider an "bf-dotnet-integration" team for these specific integration packages.

Reply:
I created a **bf-dotnet-integration** team yesterday which consists of Gary and John. I'll update the PR to include this team as well. 

@gabog, @cleemullins it's not redundant as the codeowners operates with "last write wins" semantics, from the [GitHub documentation](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#example-of-a-codeowners-file):

```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2

# Order is important; the last matching pattern takes the most
# precedence. When someone opens a pull request that only
# modifies JS files, only @js-owner and not the global
# owner(s) will be requested for a review.
*.js    @js-owner
```

In regards to repeating the teams in Adapters or functional tests, I strongly agree with repetition in functional tests as PRs on a global scale can become blocked. I'm not sure I understand the logic for having break the glass scenarios in `Adapters/`.